server/homebrew.api.js

```diff
@@ -48,6 +48,15 @@ const api = {
 			}
 			id = id.slice(googleId.length);
 		}
+
+		// ID Validation Checks
+		if(!id.match(/^[A-Za-z0-9_-]{7,14}$/)){
+			throw { name: 'ID Error', message: 'Invalid ID', status: 404, HBErrorCode: '11', brewId: id };
+		}
+		if(googleId && !googleId.match(/^1(?:[A-Za-z0-9+\/]{32}|[A-Za-z0-9+\/]{43})$/)){
```

Comment:
I do have very slight wariness on specifying exact character lengths here for Google. I guess if it changes we would know pretty quickly with a bunch of user reports. But I'm thinking maybe we don't want to subject ourselves to a panic moment like that if we just loosen this up a little to just > 33 chars.

I will let you make the call on that.